<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MethodEditClass</class>
 <widget class="QWidget" name="MethodEditClass">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>670</width>
    <height>507</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MethodEdit</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="7" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_introduction"/>
   </item>
   <item row="8" column="2">
    <widget class="QLabel" name="label_3">
     <property name="text">
      <string>Output</string>
     </property>
    </widget>
   </item>
   <item row="4" column="0">
    <widget class="QLabel" name="label_4">
     <property name="text">
      <string>Introduction</string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Procedure</string>
     </property>
    </widget>
   </item>
   <item row="10" column="0" colspan="3">
    <widget class="QPushButton" name="pushButton">
     <property name="text">
      <string>Save</string>
     </property>
    </widget>
   </item>
   <item row="9" column="2">
    <widget class="QTextEdit" name="textEdit_out"/>
   </item>
   <item row="2" column="0">
    <widget class="QLabel" name="label_5">
     <property name="text">
      <string>Name</string>
     </property>
    </widget>
   </item>
   <item row="8" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Input</string>
     </property>
    </widget>
   </item>
   <item row="9" column="0">
    <widget class="QTextEdit" name="textEdit_input"/>
   </item>
   <item row="9" column="1">
    <widget class="QTextEdit" name="textEdit_procedure"/>
   </item>
   <item row="3" column="0" colspan="3">
    <widget class="QTextEdit" name="textEdit_name"/>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
